tests/test_snapshots/test_claims_snapshot.py

```diff
@@ -0,0 +1,243 @@
+import math
+from decimal import Decimal
+
+import pytest
+import responses
+from pytest import approx
+
+from badger_api.requests import badger_api
+from helpers.constants import (
+    ARB_BADGER,
+    BADGER,
+    BOOST_CHAINS,
+    CVX_CRV_ADDRESS,
+    DIGG,
+    NETWORK_TO_BADGER_TOKEN,
+    POLY_BADGER,
+    POLY_SUSHI,
+    SWAPR_WETH_SWAPR_ARB_ADDRESS,
+    XSUSHI,
+)
+from helpers.digg_utils import digg_utils
+from helpers.enums import BalanceType, Network
+from rewards.snapshot.claims_snapshot import claims_snapshot, claims_snapshot_usd
+
+BADGER_PRICE = 27.411460272851376
+XSUSHI_PRICE = 1.201460272851376
+CVX_CRV_PRICE = 12.411460272851376
+SWAPR_WETH_SWAPR_PRICE = 12312.201460272851376
+
+TEST_WALLET = '0xD27E9195aA35A7dE31513656AD5d4D29268f94eC'
+TEST_WALLET_ANOTHER = '0xF9e11762d522ea29Dd78178c9BAf83b7B093aacc'
+
+CLAIMABLE_BALANCES_DATA_ETH = {
+    'maxPage': 1,
+    'rewards': {
+        TEST_WALLET: [
+            {
+                'network': 'ethereum', 'address': BADGER,
+                'balance': '148480869281534217908'},
```

Comment:
this is formatted weird, is formatter installed?

Reply:
I don't use autoformatter, but this is definitely PEP8 compatible. If it was a style problem I will see it by code analyzer. But I can fix that if this is a problem @btcookies ?

Reply:
If you npm install it'll run husky and black on commit

Reply:
Fixed